cmake/cvars.cmake

```diff
@@ -20,6 +20,26 @@ set(CVAR_GAME_OVERLAY_FONT "gOverlayFont" CACHE STRING "")
 set(CVAR_MENU_BAR_OPEN "gOpenMenuBar" CACHE STRING "")
 set(CVAR_PREFIX_CONTROLLERS "gControllers" CACHE STRING "")
 set(CVAR_PREFIX_ADVANCED_RESOLUTION "gAdvancedResolution" CACHE STRING "")
+
+set(LUS_BTN_A KbScancode::LUS_KB_X CACHE STRING "")
+set(LUS_BTN_B KbScancode::LUS_KB_C CACHE STRING "")
+set(LUS_BTN_L KbScancode::LUS_KB_E CACHE STRING "")
+set(LUS_BTN_R KbScancode::LUS_KB_R CACHE STRING "")
+set(LUS_BTN_Z KbScancode::LUS_KB_Z CACHE STRING "")
+set(LUS_BTN_START KbScancode::LUS_KB_SPACE CACHE STRING "")
+set(LUS_BTN_STICKUP LUS_KB_W CACHE STRING "")
```

Comment:
feels slightly odd to have some with `KbScancode::` and some without. it seems that's leftover from before (based on changes in `AxisDirectionMappingFactory`) - but this feels like a nice opportunity to add some consistency